api/package.json

```diff
@@ -12,6 +12,7 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "format": "prettier --config ../.prettierrc --ignore-path .prettierignore --write .",
+    "format:check": "prettier --config ../.prettierrc --ignore-path .prettierignore --check .",
```

Comment:
This can be removed

Reply:
My bad I was trying out git rebase and confused myself so I think I'll stick to merging master locally now 😂

Reply:
lol